_inc/search/components/api.js

```diff
@@ -0,0 +1,14 @@
+class JetpackSearchAPI {
+	getAPIUrl( query ) {
+		let url = 'https://public-api.wordpress.com/rest/v1.3/sites/20115252/search?q=';
+		url += encodeURIComponent( query );
+		url += '&fields=author';
+		return url;
+	}
+
+	fetch( query ) {
+		return window.fetch( this.getAPIUrl( query ) );
```

Comment:
Remember that `window.fetch` isn't available in some browsers, notably IE11. You'll need to polyfill it.

Reply:
In that vein, I'd recommend ponyfilling with a library like [unfetch](https://www.npmjs.com/package/unfetch)  which consumes only [554 bytes](https://bundlephobia.com/result?p=unfetch@4.1.0) gzipped.